keep-ui/app/incidents/incident-pagination.tsx

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
TODO: apply to incident alerts

Reply:
Do we expect a single incident to have so many alerts?

Reply:
For sure :)